projects/origin-release/base/Dockerfile

```diff
@@ -0,0 +1,17 @@
+#
+# This is the image that controls the base dependencies for
+# the standard build environment for releasing OpenShift Origin.
+#
+# The standard name for this image is openshift/origin-release:base
+#
+FROM        centos:7
+MAINTAINER  Steve Kuznetsov <skuznets@redhat.com>
+
+COPY cbs-paas7-openshift-multiarch-el7-build.repo /etc/yum.repos.d/
+RUN INSTALL_PKGS="bc bind-utils bsdtar ceph-common createrepo device-mapper device-mapper-persistent-data e2fsprogs epel-release ethtool file findutils gcc git glibc-static gpgme gpgme-devel hostname iptables jq krb5-devel libassuan libassuan-devel lsof make mercurial nmap-ncat openssl protobuf-compiler rsync socat sysvinit-tools tar tito tree util-linux wget which xfsprogs zip" && \
```

Comment:
This is currently the union of `openshift/base` and what used to be in the `openshift/origin-release` install line. We need EPEL for `jq` and `tito`